index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=inter' rel='stylesheet'>
+    <link rel="stylesheet" href="./style.css">
+    <title>Porfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo-container">
+            <h2>My Logo</h2>
+        </div>
+        <div class="open_mobile_menu_icon">
+            <img src="Icons/Menu.svg" alt="">
+        </div>
+    </header>
+    <section class="headline">
+        <div class="welcome_text">
+            <h2>Hey There.<br> I'm Christian.<br><span>I’m a software developer</h2></span>  
+        </div>
+        <div class="introduction_text">
+            <p class="">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="social_media_icons">
+            <div><a href="https://twitter.com/Momo_yde"><img src="Icons/twitter.svg" alt=""></a></div>
+            <div><a href="https://www.linkedin.com/in/christian-momo/"><img src="Icons/Linkedin.svg" alt=""></a></div>
+            <div><a href=""><img src="Icons/medium.svg" alt=""></a></div>
+            <div><a href="https://github.com/Momo-87"><img src="Icons/github.svg" alt=""></a></div>
+            <div><a href=""><img src="Icons/angellist.svg" alt=""></a></div>
+        </div>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (a tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly work on it. :+1: